apps/emails-and-messages/src/modules/sendgrid/configuration/sendgrid-configuration.router.ts

```diff
@@ -68,7 +68,7 @@ export const sendgridConfigurationRouter = router({
       try {
         return ctx.sendgridConfigurationService.getConfiguration(input);
       } catch (e) {
-        throwTrpcErrorFromConfigurationServiceError(e);
+        return throwTrpcErrorFromConfigurationServiceError(e);
```

Comment:
Without this change, tRPC assumed the route can return `undefined`. `throw...` function has return type `never`, so only the "happy path" is used as the return type in the "onSuccess" callback in useQuery hooks.